content/docs/user-guide/packaging/project-export/project-export-cli.md

```diff
@@ -17,6 +17,24 @@ To make best use of the `export-project` command, it is recommended to have a pr
 To learn more about the AssetBundler and Seed Files, please visit the [overview page on the AssetBundler tool](https://docs.o3de.org/docs/user-guide/packaging/asset-bundler/overview/).
 
 ## Getting Started
+### TL;DR
+For any new project that you quickly want to test with export, assuming you are using the Default Level supplied with the project as your base, you can quickly do this by double clicking on the [`export.bat`](https://github.com/o3de/o3de/blob/480ce34ab7f3056113e17056125c89ae220bef7a/Templates/DefaultProject/Template/export.bat) script (or running the [`export.sh`](https://github.com/o3de/o3de/blob/480ce34ab7f3056113e17056125c89ae220bef7a/Templates/DefaultProject/Template/export.sh) script on unix) in the root of your Project folder.
+
+Project Export functionality is also available on a per-project basis. This is because there are helper export scripts that are included in the Project Templates, as can be seen with. These scripts are included at the root folder of every new Project created with the `DefaultProject` and `MinimalProject` templates.
+
+The package scripts pre-populate the relevant arguments for the given project, and then call `export-project` on the user's behalf.
+
+If you need to run it from the command line, simply run the following command (assuming project has already been created and registered):
+```
+# On Windows
+\path\to\project\export.bat
+
+#On Linux
+\path\to\project\export.sh
+```
+Please note that this will only cover the most basic use cases for getting started quickly. For more info on the export-command in general, keep reading below.
```

Comment:
{{< note >}}
Be sure your [project is registered with the engine](https://youtu.be/SZC13S0YZZs?si=zlkAnflTiHQCYcA2&t=146).
{{< /note >}}

{{< note >}}
Export.bat will only cover the most basic use cases for getting started. For more info on the export-command in general, keep reading below.
{{< /note >}}